pkg/server/server.go

```diff
@@ -308,6 +308,7 @@ func (s *Server) HTTPHandler() http.Handler {
 	handle("/api/helm/template", authHandlerWithUser(helmConfig.HandleHelmRenderManifests))
 	handle("/api/helm/releases", authHandlerWithUser(helmConfig.HandleHelmList))
 	handle("/api/helm/release", authHandlerWithUser(helmConfig.HandleHelmInstall))
+	handle("/api/helm/manifests", authHandlerWithUser(helmConfig.HandleGetReleaseManifest))
```

Comment:
we would like to follow rest-api design. Hence, we do not need an extra endpoint here, we should reuse `/api/helm/release`, so that:
* `POST /api/helm/release` install a new releases
* `GET /api/helm/release` retrieves the existing release